examples/terminal/commands/interact.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+
+import cmd2
+from examples.terminal.core.state import CFState
+from counterfit.core.output import CFPrint
+from counterfit.core.targets import CFTarget
+
+
+def get_targets():
+    targets = []
+    for target_name, target_obj in sorted(CFState.state().get_targets().items()):
+        targets.append(target_name)
+    return targets
+
+
+def interact_cmd(args: argparse.Namespace) -> None:
+    target_name = args.target
+    try:
+        target: CFTarget = CFState.state().get_targets().get(target_name, None)
+        if not target:
+            CFPrint.failed(f"Could not load target {target_name}")
+        target.load()
```

Comment:
Could you clean-up some of the earlier version command modules, for example, `interact` has been replaced with `set_target`. So `interact` module can be deleted.

Reply:
@rdheekonda, let's have a quick chat about this tomorrow 👍